js/script.js

```diff
@@ -0,0 +1,125 @@
+  },
+  {
+    image: './assets/doctor3.jpg',
+    name: 'Dr. Anna Walker',
+    organization: 'Traumatologist',
+    description:
+      "As the author of <Digital Art Art of Our Time>, he opened 'Art Center Nabi', Korea's first digital art institution in 2000, and is currently serving.",
+  },
+  {
+    image: './assets/doctor2.jpg',
+    name: 'Dr. Paul Ford',
+    organization: 'Cardiac Surgeon',
+    description:
+      "By developing Asia's first Internet protocol network SDN and leading Korea's first private line Internet connection in 1990, it ushered in the era of the Internet in earnest.",
+  },
+
+  {
+    image: './assets/doctor5.jpg',
+    name: 'Dr. Sarah Snow',
+    organization: 'Pediatrician',
+    description:
+      "Layla Tretikov is the general secretary of the Wikimedia Foundation, a non-profit organization that runs Wikipedia. Wikipedia is provided free of charge in 290 languages ​​every month to over 100 million people, nearly half of the world's population.",
+  },
+  {
+    image: './assets/doctor4.jpg',
+    name: 'Dr. Andrew Green',
+    organization: 'Dermatologist',
+    description:
+      "European integration and online youth participation in politics and democracy are major concerns, and a report has been published that will potentially affect the revision of the EU's copyright law in July.",
+  },
+
+  {
+    image: './assets/doctor6.jpg',
+    name: 'Dr. Sandra Wok',
+    organization: 'Gynicologist',
+    description:
+      'He led open source projects at the Mozilla Foundation and joined CC as CEO in 2014. He has been active in open movements such as open government and open source.',
+  },
+];
+
+const showSpeakers = [speakers[0], speakers[1]];
+
+const body = document.querySelector('body');
+const speakersSection = document.createElement('div');
+
+speakersSection.setAttribute('class', 'section speakers');
+
+const speakersSectionTitle = document.createElement('div');
+speakersSectionTitle.setAttribute('class', 'title speakers-title');
+speakersSectionTitle.innerHTML = '<span>Featred speakers</span>';
+
+const speakersSectionContent = document.createElement('div');
+speakersSectionContent.setAttribute('class', 'content speakers-content');
+
+function AppendSpeakerItem(Speakers) {
+  // eslint-disable-next-line no-restricted-syntax
+  for (const element of Speakers) {
+    const speakerItem = document.createElement('div');
+    speakerItem.setAttribute('class', 'speaker-item');
+    speakerItem.innerHTML = `<img class="speaker-item-img" src=${element.image} alt="speaker1" />
+
+          <h4 class="speaker-name">${element.name}</h4>
+          <p class="speaker-organize">${element.organization}</p>
+          <hr />
+          <p class="speaker-desc">
+           ${element.description}
+          </p>`;
+
+    speakersSectionContent.append(speakerItem);
+  }
+}
```

Comment:
- [ ] Great job working on the js and adding the speaker dynamically as required but please try to adjust the spaces to match the original design since I can see the paragraph in under the image just keep it beside the image as the original design 

- Your design
![image](https://user-images.githubusercontent.com/58553711/197083437-4e73d583-08af-4330-ae33-cc26663c723e.png)

 - Original design
 
![image](https://user-images.githubusercontent.com/58553711/197083495-5557db2f-5673-4db1-8c5a-31a288f788d0.png)

Reply:
Hi @Reem-lab,
Thank you for your feedback and for recognizing the work done on the JavaScript and dynamic speaker addition.
I will make sure to adjust the spacing to match the original design, specifically in regards to the paragraph positioning next to the image.